src/classes/TDTM_TriggerHandler.cls

```diff
@@ -242,4 +245,20 @@ public class TDTM_TriggerHandler {
                     
         return errors;
     }
+
+    /*******************************************************************************************************
+    * @description Modifies the global static collection of Trigger_Handler records to disable or enable
+    * a specific TriggerHander by SObjectName and ApexClass name.
+    * @param objName SObject Name
+    * @param className Apex Class Name
+    * @param state True (enable) | False (disable)
+    */
+    public static void toggleTriggerState(String objName, String className, Boolean state) {
```

Comment:
is this truly needed, or only used in places where you could have used existing flags/counters that were in use?  I'd prefer avoiding two different methodologies for tackling these issues if we can avoid it.